cmd/argo/commands/executorplugin/io.go

```diff
@@ -0,0 +1,90 @@
+package executorplugin
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"text/template"
+
+	apiv1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/yaml"
+
+	"github.com/argoproj/argo-workflows/v3/pkg/plugins/spec"
+)
+
+func loadPluginManifest(pluginDir string) (*spec.Plugin, error) {
+	manifest, err := os.ReadFile(filepath.Join(pluginDir, "plugin.yaml"))
```

Comment:
Not sure if I'm a huge fan of hardwiring `plugin.yaml`. It makes sense for things like `Dockerfile`, but `plugin.yaml` is too generic. At the very least it should be a customizable argument, but I would argue it's better to require file name explicitly.

Reply:
`Plugin`?